runtime/wasm/src/module/test.rs

```diff
@@ -377,20 +377,17 @@ async fn ipfs_map() {
     let ipfs = Arc::new(ipfs_api::IpfsClient::default());
```

Comment:
Did this file have to change at all?

Reply:
These changes are unrelated, I had issues running the tests locally which apparently resolved with this changes.